docs/References.bib

```diff
@@ -1,3 +1,18 @@
+@article{Lindblom2005qh,
+      author         = "Lindblom, Lee and Scheel, Mark A. and Kidder,
+                        Lawrence E. and Owen, Robert and Rinne, Oliver",
+      title          = "{A New generalized harmonic evolution system}",
```

Comment:
This capitalization is ridiculous, but since it's due to a bad record in INSPIRE HEP I'll let it through.